packages/catalog-search/src/SearchSuggestions.jsx

```diff
@@ -11,17 +13,24 @@ const SearchSuggestions = ({
   handleSuggestionClickSubmit,
   disableSuggestionRedirect,
 }) => {
-  const getLinkToCourse = (course) => `/${enterpriseSlug}/course/${course.key}`;
+  const getLinkToCourse = (course) => {
+    const { learning_type: learningType } = course;
+    if (learningType === LEARNING_TYPE_EXECUTIVE_EDUCATION) {
+      return `/${enterpriseSlug}/executive-education-2u/course/${course.key}`;
+    }
+
+    return `/${enterpriseSlug}/course/${course.key}`;
```

Comment:
I think all the links have the course_type ... `/${enterpriseSlug}/**course_type**/course/${course.key}`
reference:https://github.com/openedx/frontend-app-learner-portal-enterprise/blob/master/docs/decisions/0008-externally-hosted-course-enrollments.md

Reply:
@irfanuddinahmad I feel it would better to do this in a separate PR. Maybe file a ticket so that we can change this all other links to the updated version and also have enough time to properly test it out.

Reply:
@irfanuddinahmad @saleem-latif I think as implemented here makes sense; only externally hosted courses have a route of `/:enterpriseSlug/:courseType/course/:courseKey`. Normal OCM courses should have the standard route of `/:enterpriseSlug/course/:courseKey`.